### PR TITLE
Added Runes Skjoldr

### PR DESCRIPTION
**Pull request by vicktorhu**
Added a new keyboard, Runes Skjoldr

## Description

Added a new keyboard, Runes Skjoldr

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/runes/skjoldr/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Skjöldr
+
+![Skjöldr](https://i.imgur.com/HtRqhxD.jpg)
```
Comment (drashna) [inline keyboards/runes/skjoldr/readme.md:3]:
```suggestion
![Skjöldr](https://i.imgur.com/HtRqhxDh.jpg)
```

**Review by drashna — APPROVED** [review]

File: keyboards/runes/skjoldr/config.h [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x726E
+#define PRODUCT_ID   0x736B
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Runes
+#define PRODUCT      Skjöldr
```
Comment (fauxpark) [inline keyboards/runes/skjoldr/config.h:27]: Does this work properly? Unicode characters may not work properly due to the fact that these defines aren't string literals (yet).

File: keyboards/runes/skjoldr/rules.mk [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```
Comment (fauxpark) [inline keyboards/runes/skjoldr/rules.mk:7]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/runes/skjoldr/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Reset the keyboard with **FN** + **Backspace** or by pressing the reset button on the PCB.
+  
+Make example for this keyboard (after setting up your build environment):
+
+    make skjoldr:default
+
+Flashing example for this keyboard:
+
+    make skjoldr:default:flash
```
Comment (fauxpark) [inline keyboards/runes/skjoldr/readme.md:21]:
```suggestion
    make runes/skjoldr:default

Flashing example for this keyboard:

    make runes/skjoldr:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by vicktorhu** [comment]
Wait sorry I mis-pressed the 'Resolve conversation' button.

File: keyboards/runes/skjoldr/skjoldr.h [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	K03, K13, K23, K33, K43, K53, K63, K73, K83, K93, KA3,      KC3, KD3, \
+	K04, K14, K24,           K54,                K94, KA4, KB4, KC4, KD4 \
+) { \
+	{K00, K10, K20, K30,   K40,   K50, K60,   K70,   K80,    K90, KA0, KB0,   KC0, KD0 }, \
+	{K01, K11, K21, K31,   K41,   K51, K61,   K71,   K81,    K91, KA1, KB1,   KC1, KD1 }, \
+	{K02, K12, K22, K32,   K42,   K52, K62,   K72,   K82,    K92, KA2, KB2,   KC2, KC_NO }, \
+	{K03, K13, K23, K33,   K43,   K53, K63,   K73,   K83,    K93, KA3, KC_NO, KC3, KD3 }, \
+	{K04, K14, K24, KC_NO, KC_NO, K54, KC_NO, KC_NO, KC_NO,  K94, KA4, KB4,   KC4, KD4 } \
+}
+
```
Comment (fauxpark) [inline keyboards/runes/skjoldr/skjoldr.h:42]:
```suggestion

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by vicktorhu** [comment]
Is there anything else I could fix @fauxpark? Would be glad to help.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by vicktorhu** [comment]
anything else I could help @fauxpark?

**Comment by vicktorhu** [comment]
Is there anything else I could fix @fauxpark @drashna? Would be glad to help.

**Comment by vicktorhu** [comment]
Sorry for tagging you again, @drashna @fauxpark. I would be really glad to help if there is something I could do to resolve this pull request.

**Comment by fauxpark** [comment]
You resolved my review regarding the USB product string without responding to it.

**Comment by vicktorhu** [comment]
> You resolved my review regarding the USB product string without responding to it.

Ah sorry for that! Will respond to that really quick. Thank you so much for responding.

File: keyboards/runes/skjoldr/config.h (hunk 2 above)
Comment (vicktorhu) [reply keyboards/runes/skjoldr/config.h:27]: Sorry for the super late reply. I tested this with this keyboard and my previous one ([VÆNGR](https://github.com/qmk/qmk_firmware/blob/master/keyboards/runes/vaengr/config.h)) and I don't have any problem. 

I have tested and it work fine on Windows, macOS and Linux.

It was recognized properly in linux lsusb and windows control panel:
image

Once again, really sorry for the late reply. Thought I already replied to this.

**Review by fauxpark — APPROVED** [review]

**Comment by vicktorhu** [comment]
Thank you for the merge!

**Pull request merged**